bus-mapping/src/circuit_input_builder/access.rs

```diff
@@ -145,117 +145,124 @@ pub fn gen_state_access_trace<TX>(
+                    accs.push(Access::new(i, WRITE, Account { address }));
+                    accs.push(Access::new(i, READ, Code { address }));
+                    if push_call_stack {
+                        call_stack.push((address, CodeSource::Address(address)));
+                    }
                 }
-            }
-            OpcodeId::CALLCODE => {
-                let address = contract_address;
-                accs.push(Access::new(i, WRITE, Account { address }));
+                OpcodeId::CALLCODE => {
+                    let address = contract_address;
+                    accs.push(Access::new(i, WRITE, Account { address }));
 
-                let address = step.stack.nth_last(1)?.to_address();
-                accs.push(Access::new(i, WRITE, Account { address }));
-                accs.push(Access::new(i, READ, Code { address }));
-                if push_call_stack {
-                    call_stack.push((address, CodeSource::Address(address)));
+                    let address = step.stack.nth_last(1)?.to_address();
+                    accs.push(Access::new(i, WRITE, Account { address }));
+                    accs.push(Access::new(i, READ, Code { address }));
+                    if push_call_stack {
+                        call_stack.push((address, CodeSource::Address(address)));
+                    }
                 }
-            }
-            OpcodeId::DELEGATECALL => {
-                let address = step.stack.nth_last(1)?.to_address();
-                accs.push(Access::new(i, READ, Code { address }));
-                if push_call_stack {
-                    call_stack.push((contract_address, CodeSource::Address(address)));
+                OpcodeId::DELEGATECALL => {
+                    let address = step.stack.nth_last(1)?.to_address();
+                    accs.push(Access::new(i, READ, Code { address }));
+                    if push_call_stack {
+                        call_stack.push((contract_address, CodeSource::Address(address)));
+                    }
                 }
-            }
-            OpcodeId::STATICCALL => {
-                let address = step.stack.nth_last(1)?.to_address();
-                accs.push(Access::new(i, READ, Code { address }));
-                if push_call_stack {
-                    call_stack.push((address, CodeSource::Address(address)));
+                OpcodeId::STATICCALL => {
+                    let address = step.stack.nth_last(1)?.to_address();
+                    accs.push(Access::new(i, READ, Code { address }));
+                    if push_call_stack {
+                        call_stack.push((address, CodeSource::Address(address)));
+                    }
                 }
+                _ => {}
             }
-            _ => {}
+            Ok(())
+        })();
+        if let Err(e) = result {
+            log::warn!("err when parsing access: {:?}, step {:?}", e, step);
```

Comment:
Wdyt if we log + break here earlier to skip all the left trace parsing? Although it wont happened on a `normal` geth trace since this op should be the last op, but error handling perspective terminate here make more sense

Reply:
I c, just realized stack underflow wont stop depth - 1 execution